pkg/database/user_report_test.go

```diff
@@ -124,21 +124,49 @@ func TestDeleteUserReport(t *testing.T) {
 	t.Parallel()
 
 	db, _ := testDatabaseInstance.NewDatabase(t, nil)
+	realm, err := db.FindRealm(1)
+	if err != nil {
+		t.Fatal(err)
+	}
+	realm.AddUserReportToAllowedTestTypes()
+	if err := db.SaveRealm(realm, SystemTest); err != nil {
+		t.Fatal(err)
+	}
 
 	phoneNumber := "+8128675309"
-	userReport, err := db.NewUserReport(phoneNumber, generateNonce(t), true)
-	if err != nil {
-		t.Fatalf("error creating user report: %v", err)
+	verCode := VerificationCode{
```

Comment:
```suggestion
	verCode := &VerificationCode{
```

We usually create a pointer in tests instead of taking a pointer later.